src/applications/mhv-secure-messaging/containers/EditContactList.jsx

```diff
@@ -81,23 +88,26 @@ const EditContactList = () => {
 
   const handleSaveAndExit = async (e, forceSave) => {
     e.preventDefault();
+    if (!isMinimumSelected) {
+      setMinimumSelectError(ErrorMessages.ContactList.MINIMUM_SELECTION);
+      focusOnErrorField();

```

Comment:
as we discussed, this focus handler is redundant with the one in `useEffect` on line 160. I suggest either 
- adding `await` to `setMinimumSelectError` on line above and remove `useEffect` on line 157 or
- remove `focusOnErrorField` on line 93 and keep `useEffect`